proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,393 @@
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.MountTopicsResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc UnmountTopics(UnmountTopicsRequest) returns (UnmountTopicsResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha2/cloud-storage/topics/unmount"
+      body: "topics"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Unmount topics to tiered storage"
+      description: "This operation offloads topics to tiered storage, freeing up all local cluster resources. Once a topic is unmounted, it can no longer be consumed or produced, effectively disappearing from the active cluster while its data remains safely stored in the external tiered storage."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.UnmountTopicsResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc ListMountableTopics(ListMountableTopicsRequest) returns (ListMountableTopicsResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha2/cloud-storage/topics/mountable"
+      response_body: "topics"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Retrieve a list of mountable topics"
+      description: "This operation retrieves all topics that are currently unmounted and available to be mounted to the cluster. These topics reside in cloud storage and can be mounted for consumption or production within the cluster."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.ListMountableTopicsResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc ListMountTasks(ListMountTasksRequest) returns (ListMountTasksResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha2/cloud-storage/mount-tasks"
+      response_body: "tasks"
```

Comment:
Just a note that we say mount/unmount "operation" in the [main doc](https://deploy-preview-725--redpanda-docs-preview.netlify.app/24.3/manage/mountable-topics/#monitor-progress). I guess you could also say that a mount/unmount might consist of many tasks?

Reply:
You can run multiple mount or unmount operations (each containing one or more topics) at the same time. Not sure I got your question

Reply:
@weeco Sorry about that - I just meant to note that in the main doc we ended up saying "mount operation" instead of "mount task"